storybook/storybook-css/src/components/Heading/Heading.stories.tsx

```diff
@@ -10,26 +10,21 @@ import '@amsterdam/design-system-css/src/heading/heading.scss'
 const meta = {
   title: 'CSS Components/Heading',
   component: Heading,
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+    level: 1,
+    size: 'level-1',
+  },
   argTypes: {
     level: {
       table: {
         disable: true,
       },
     },
-    styleLevel: {
-      options: [1, 2, 3, 4],
-      control: 'select',
-      table: {
-        type: { summary: 'number' },
-        defaultValue: { summary: '1' },
-      },
+    size: {
+      options: ['level-1', 'level-2', 'level-3', 'level-4'],
```

Comment:
Don't we want to keep this a `select`?

Reply:
I think I prefer radio buttons for about five options or less – we have four here. Saves the user a click and clearly shows the options up front.

We should create a Storybook guideline for this.